### ISSUE TITLE
[idea], [proofofconcept] big area for clicking/touching seekbar

### ISSUE THREAD

**Issue by kocoten1992** [videojs/video.js#4710]
### Description
Our seekbar click area is a bit small (3px normal, 5px on hover), user got a hard time try to click on seekbar.

YT got the same thing, but they add a hidden layer ```ytp-progress-bar-padding``` that got height of 22px when hover (16px normal), user don't miss click when there is a layer like that.

![ytpadding](https://user-images.githubusercontent.com/7130705/32276626-5abe1c12-bf42-11e7-9f4f-09629e30655e.png)

Here is my hack try to achieve that:

```js
var SeekBar = videojs.getComponent('SeekBar')

class MySeekBar extends SeekBar {
  createEl() {

    var seekBarEl = super.createEl('div', {
      className: 'vjs-progress-holder'
    }, {
      'aria-label': this.localize('Progress Bar')
    });

    var paddingEl = super.createEl('div', {
      className: 'vjs-progress-padding' // named after yt
    })

    seekBarEl.appendChild(paddingEl)
  
    return seekBarEl
  }

videojs.registerComponent('SeekBar', MySeekBar)
}
```

```css
.vjs-progress-padding {
  margin: unset;
  height: 22px;
  bottom: 10px;
}
```

Any thought ?

**Comment by gkatsev** [comment]
Yeah, it's something we should add. Adding a padding like this could potentially create issues with styling.
I've previously added scrubbing support for this in https://github.com/videojs/video.js/pull/4004 but at that time I guess I forgot to add click-to-seek support and haven't had a change to come back and implement that.
